tests/task.test.ts

```diff
@@ -0,0 +1,23 @@
+import { taskReducer, TaskState, SetCurrentTaskAction, SET_CURRENT_TASK } from '../src/task';
+
+describe('task reducer', () => {
+  test('should return initial state', () => {
+    const expected = {};
+
+    const initialState = undefined;
+    const result = taskReducer(initialState, {} as SetCurrentTaskAction);
+
+    expect(result).toStrictEqual(expected);
+  });
+
+  test('should handle SET_CURRENT_TASK action', () => {
```

Comment:
Can we phrase the test names as 'when <CONDITION> then <ACTION>' so we know exactly what is being tested. This test name doesn't tell us what is actually expected to happen when a SET_CURRENT_TASK action is receieved